templates/openstack/support.html

```diff
@@ -0,0 +1,530 @@
+            </div>
+            <div class="col-4 u-hide--medium u-hide--small">
+              {{ image (
+                url="https://assets.ubuntu.com/v1/0cfe6c2d-build.svg",
+                alt="Ceph managed deploy",
+                width="134",
+                height="88",
+                hi_def=True,
+                loading="lazy",
+                ) | safe
+              }}
+            </div>
+          </div>
+        </p>
+      </li>
+
+      <li class="p-stepped-list__item">
+        <h3 class="p-stepped-list__title">Operate</h3>
+        <p class="p-stepped-list__content">
+          <div class="row u-no-padding">
+            <div class="col-8">
+              <p>After day-1, day-2 comes in. The cloud can either be managed by you or by our team of cloud experts. In both cases, we provide 24x7 enterprise-grade support, including security updates, phone and ticket support, local regulatory compliance programmes, and the like.</p>
+            </div>
+            <div class="col-4 u-hide--medium u-hide--small">
+              {{
+                  image(
+                  url="https://assets.ubuntu.com/v1/81e3bf86-We+operate+your+Kubernetes.svg",
+                  alt="",
+                  width="100",
+                  height="100",
+                  hi_def=True,
+                  loading="lazy",
+                  ) | safe
+                }}
+            </div>
+          </div>
+        </p>
+      </li>
+
+      <li class="p-stepped-list__item">
+        <h3 class="p-stepped-list__title">Transfer</h3>
+        <p class="p-stepped-list__content">
+          <div class="row u-no-padding">
+            <div class="col-8">
+              <p>Some customers choose to extend their support with fully managed service. On request, we transfer control over your cloud back to your organisation. We train your personnel, make sure you are able to operate the cloud yourself moving forward and perform a formal handover process. We continue providing commercial support to make sure your cloud receives security updates and you receive help if needed.</p>
+            </div>
+            <div class="col-4 u-hide--medium u-hide--small">
+              {{ image (
+                url="https://assets.ubuntu.com/v1/2131b805-We+transfer+control.svg",
+                alt="Ceph managed transfer",
+                width="134",
+                height="134",
+                hi_def=True,
+                loading="lazy",
+                ) | safe
+              }}
+            </div>
+          </div>
+        </p>
+      </li>
+    </ol>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="u-fixed-width">
+    <h2>Ubuntu Advantage pricing</h2>
+  </div>
+  {% with no_desktop="true" %}{% include "shared/pricing/_ua-for-infrastructure.html" %}{% endwith %}
```

Comment:
it looks like there is a small difference between this table and the doc, in the row 'Phone and ticket support' under 'Essential', it just has a dash

Reply:
We're using that partial in a few places, and the meaning is the same, so I'll leave that as is.